.github/workflows/release.yml

```diff
@@ -41,8 +41,9 @@ jobs:
       with:
         output-dir: dist
       env:
-        CIBW_BUILD: "cp37-* cp38-* cp39-* cp310-*"
-        CIBW_ARCHS_MACOS: x86_64 universal2
+        CIBW_BUILD: "cp38-* cp39-* cp310-* cp311-*"
+        CIBW_ARCHS_MACOS: x86_64 arm64
+        CIBW_ENVIRONMENT_MACOS: "CFLAGS='-arch arm64 -arch x86_64' CXXFLAGS='-arch arm64 -arch x86_64' LDFLAGS='-arch arm64 -arch x86_64'"
```

Comment:
why did you remove `universal2` from `CIBW_ARCHS_MACOS` at the same time as `CIBW_ENVIRONMENT_MACOS` contains both architecture flags `-arch arm64 -arch x86_64`?

Reply:
> why did you remove universal2 from CIBW_ARCHS_MACOS

Initially just to diagnose why the `arm64` part of the `universal2` wheel was broken (it has been since the initial introduction of that wheel, but until recently was something I was unable to check).

> CIBW_ENVIRONMENT_MACOS contains both architecture flags -arch arm64 -arch x86_64

Not having `-arch arm64` C/CXX/LDFLAGS set resulted in the `arm64` binary building incorrectly. So you could install the wheel, but when you went to use `pyots` it would not function correctly. Turns out the ENVIRONMENT flags do not control what binary gets built (the `CIBW_ARCHS_MACOS` and related settings do that), but the flags need to be set when the architecture in question is being built (for `arm64` at least). If you check the resulting binaries (.so files in the package) with e.g. `file` or `lipo` you can see that the `x86_64` version contains only `x86_64` and the `arm64` one contains only `arm64`. For CIBW there's only one opportunity to set the ENVIRONMENT flags, so I set both because I'm building both wheels, and everything now works as expected.

I was initially planning to reset to `universal2` once I got the `arm64` situation figured out, but after some thought that seemed wasteful (the wheel is ~2x the size of `x86_64` or `arm64` individual builds) so I just left it as independent. Do you have a need for the `universal2` wheel?